docs/_core/theming/intro.story.tsx

```diff
@@ -0,0 +1,103 @@
+import { Source } from '@storybook/components';
+import React from 'react';
+
+export const Intro = () => (
+  <>
+    <p>
+      Castor comes with generated theme <code>.css</code> files. You can use
+      them directly in your app.
+    </p>
+    <p>For example, you may import a &#34;day&#34; theme via JavaScript:</p>
+    <Source
+      language="js"
+      code={`
+        import '@onfido/castor/dist/themes/day.css';
+      `}
+      format
+      dark
+    />
+    <p>
+      Then if you import also a &#34;night&#34; theme, it will override any
+      previously imported one:
+    </p>
+    <Source
+      language="js"
+      code={`
+        import '@onfido/castor/dist/themes/night.css';
+      `}
+      format
+      dark
+    />
+    <p>
+      This happens because a theme (or theme CSS variables, to be more precise)
+      by default gets attached to a <code>:root</code> element of a document. So
+      there can only be one at a time.
+    </p>
+    <p>
+      In order to be able to switch between themes, Castor also comes with
+      generated &#34;classed&#34; version of themes, alongside a theme switching
```

Comment:
💅🏻 

```suggestion
      generated &#34;class&#34; version of themes, alongside a theme switching
```

Reply:
This defines "what kind of themes", that's why it's in quotes, same as: https://github.com/onfido/castor/tree/main/packages/core#switch-theme

Reply:
I know, I'm suggesting changing `classed` to `class`.

Why do we need the character html codes btw, can't we use `"`?

Reply:
There's linting rule enabled:

![Screenshot 2021-03-02 at 12 54 12](https://user-images.githubusercontent.com/20243687/109651406-65c8e280-7b56-11eb-8afd-d09b74b8987e.png)